features/distro-resources/src/main/resources/items/demo.items

```diff
@@ -26,77 +26,77 @@ Group:Number:AVG                Temperature "Avg. Room Temperature [%.1f °C]"
 Group:Contact:OR(OPEN, CLOSED)  Windows     "Open windows [(%d)]"               <contact>
 
 /* Lights */
-Dimmer Light_GF_Living_Table        "Table"         (GF_Living, Lights)     ["Lighting"]
-Switch Light_GF_Corridor_Ceiling    "Ceiling"       (GF_Corridor, Lights)
-Switch Light_GF_Kitchen_Ceiling     "Ceiling"       (GF_Kitchen, Lights)
-Switch Light_GF_Kitchen_Table       "Table"         (GF_Kitchen, Lights)
-Switch Light_GF_Corridor_Wardrobe   "Wardrobe"      (GF_Corridor, Lights)
-Switch Light_GF_Toilet_Ceiling      "Ceiling"       (GF_Toilet, Lights)
-Switch Light_GF_Toilet_Mirror       "Mirror"        (GF_Toilet, Lights)
-
-Switch Light_FF_Bath_Ceiling        "Ceiling"       (FF_Bath, Lights)
-Switch Light_FF_Bath_Mirror         "Mirror"        (FF_Bath, Lights)
-Switch Light_FF_Corridor_Ceiling    "Corridor"      (FF_Corridor, Lights)
-Switch Light_FF_Office_Ceiling      "Ceiling"       (FF_Office, Lights)
-Switch Light_FF_Son_Ceiling         "Ceiling"       (FF_Son, Lights)
-Switch Light_FF_Daughter_Ceiling    "Ceiling"       (FF_Daughter, Lights)
-Switch Light_FF_Bed_Ceiling         "Ceiling"       (FF_Bed, Lights)
-
-Switch Light_C_Corridor_Ceiling     "Ceiling"       (gC, Lights)
-Switch Light_C_Staircase            "Staircase"     (gC, Lights)
-Switch Light_C_Washing_Ceiling      "Washing"       (gC, Lights)
-Switch Light_C_Workshop             "Workshop"      (gC, Lights)            ["Lighting"]
```

Comment:
We should be careful with removing "Lighting" - this has been used so far for Alexa, Homekit, etc.
Maybe just leave it for now as a valid option - just like you also kept "CurrentTemperature" below.

Reply:
Sorry, I thought the same thing when further down the file and forgot those...!